homeassistant/components/flexpool/const.py

```diff
@@ -0,0 +1,57 @@
+"""Constants for the Flexpool integration."""
+
+DOMAIN = "flexpool"
+
+SENSOR_DICT = {
+    "flexpool_unpaid_balance": [
+        "Flexpool Unpaid Balance",
+        "mdi:wallet",
+    ],
```

Comment:
Instead of a list, you could use a dictionary, that way it would be more clear what each item is. E.g.,:

```py
"flexpool_current_reported": {
    CONF_NAME: "Flexpool Current Reported Hashrate",
    CONF_ICON: "mdi:pound",
    CONF_??: "reported",
},
```

Or something similar...